src/auth/project-config.ts

```diff
@@ -122,16 +141,17 @@ export class ProjectConfig {
    */
   public static buildServerRequest(configOptions: UpdateProjectConfigRequest): ProjectConfigClientRequest {
     ProjectConfig.validate(configOptions);
-    const request = configOptions as any;
-    if (configOptions.multiFactorConfig !== undefined) {
+    const request: ProjectConfigClientRequest = {};
+    if (typeof configOptions.smsRegionConfig !== 'undefined') {
+      request.smsRegionConfig = configOptions.smsRegionConfig;
+    }
+    if (typeof configOptions.multiFactorConfig !== 'undefined') {
       request.mfa = MultiFactorAuthConfig.buildServerRequest(configOptions.multiFactorConfig);
     }
-    // Backend API returns "mfa" in case of project config and "mfaConfig" in case of tenant config.
-    // The SDK exposes it as multiFactorConfig always. 
-    // See https://cloud.google.com/identity-platform/docs/reference/rest/v2/projects.tenants#resource:-tenant 
-    // and https://cloud.google.com/identity-platform/docs/reference/rest/v2/Config
-    delete request.multiFactorConfig;
-    return request as ProjectConfigClientRequest;
```

Comment:
Are these still valid?

Reply:
These lines have been removed because there is a conversion in multiple configs, so the server request is being built rather than JSON-ising it directly.